assets/js/components/setup/setup-wrapper.js

```diff
@@ -94,75 +81,66 @@ class SetupWrapper extends Component {
 		}, 500, 'later' );
 	}
 
-	render() {
-		const { currentModule } = this.state;
-		const setupModule = SetupWrapper.loadSetupModule( currentModule );
-		const settingsPageURL = getSiteKitAdminURL(
-			'googlesitekit-settings',
-			{}
-		);
+	const setupModule = loadSetupModule();
 
-		return (
-			<Fragment>
-				<Header />
-				<div className="googlesitekit-setup">
-					<div className="mdc-layout-grid">
-						<div className="mdc-layout-grid__inner">
-							<div className="
-								mdc-layout-grid__cell
-								mdc-layout-grid__cell--span-12
-							">
-								<section className="googlesitekit-setup__wrapper">
+	return (
+		<Fragment>
+			<Header />
+			<div className="googlesitekit-setup">
+				<div className="mdc-layout-grid">
+					<div className="mdc-layout-grid__inner">
+						<div className="
+							mdc-layout-grid__cell
+							mdc-layout-grid__cell--span-12
+						">
+							<section className="googlesitekit-setup__wrapper">
+								<div className="mdc-layout-grid">
+									<div className="mdc-layout-grid__inner">
+										<div className="
+											mdc-layout-grid__cell
+											mdc-layout-grid__cell--span-12
+										">
+											<p className="
+												googlesitekit-setup__intro-title
+												googlesitekit-overline
+											">
+												{ __( 'Connect Service', 'google-site-kit' ) }
+											</p>
+											{ setupModule }
+										</div>
+									</div>
+								</div>
+								<div className="googlesitekit-setup__footer">
 									<div className="mdc-layout-grid">
 										<div className="mdc-layout-grid__inner">
 											<div className="
-												mdc-layout-grid__cell
-												mdc-layout-grid__cell--span-12
-											">
-												<p className="
-													googlesitekit-setup__intro-title
-													googlesitekit-overline
+													mdc-layout-grid__cell
+													mdc-layout-grid__cell--span-2-phone
+													mdc-layout-grid__cell--span-4-tablet
+													mdc-layout-grid__cell--span-6-desktop
 												">
-													{ __( 'Connect Service', 'google-site-kit' ) }
-												</p>
-												{ setupModule }
+												<Link
+													id={ `setup-${ moduleToSetup }-cancel` }
```

Comment:
Please, use `module.slug` instead of `moduleToSetup`:

```suggestion
													id={ `setup-${ module.slug }-cancel` }
```